src/setup/demo.md

```diff
@@ -0,0 +1,236 @@
+
+```sh
+pip install substra
+# Or if a specific version is required by the team
+pip install substra==0.6.0 --force-reinstall
+
+# Check that Substra is working
+substra --version
+# For example: substra, version 0.6.0
+```
+
+You will also need to install the [Substra Tools](https://github.com/SubstraFoundation/substra-tools/tree/master/substratools) package which contains all assets templates you will need to submit your algo to the Substra network:
+
+```sh
+pip install substratools
+```
+
+### Edit your hosts file
+
+As the demo instance is a demo instance, you will have to create a link between the server ip address (provided with your credentials) and the 2 nodes domains used in the demo:
+
+```sh
+# Replace <CHANGE_ME> with the ip provided by the team
+echo "<CHANGE_ME> substra-backend.node-1.com substra-frontend.node-1.com substra-backend.node-2.com substra-frontend.node-2.com" | sudo tee -a /etc/hosts
+```
+
+This will add a line like the following to the file `/etc/hosts` of your machine:
+
+```sh
+# 01.02.003.004     substra-backend.node-1.com substra-backend.node-2.com substra-frontend.node-1.com substra-frontend.node-2.com
+```
+
+In order to check that your local substra client can reach the demo server, you can use the following command:
+
+```sh
+# Quick test on the backend of the node-1
+curl substra-backend.node-1.com/readiness
+```
+
+You should receive an `"OK"` answer, if not please check again your `/etc/hosts` file. You can also use your browser to visit [substra-frontend.node-1.com](http:substra-frontend.node-1.com) & [substra-frontend.node-2.com](http:substra-frontend.node-2.com) (without https, because this is demo).
+
+### Login
+
+You can now configure your local Substra client, *with the credentials you received from us*, to connect to the `node-1` which hosts the organisation `org-1`:
+
+```sh
+# Configure the client, with a profile, a user, a password and an url
+substra config --profile <PROFILE> -u <USER> -p "<PASSWORD>" http://substra-backend.node-1.com
+
+# Login with the profile you created (located in your home folder: ~/.substra)
+substra login --profile <PROFILE>
+
+# You can then list the network nodes that are on the Substra network
+substra list node --profile <PROFILE>
+
+# And list, for example, the registered datasets in the node you are logged in
+substra list dataset --profile <PROFILE>
+```
+
+## 2. Follow Titanic example
+
+You will find everything you need to run a first end to end example with the tabular data of the Titanic: [repository](https://github.com/SubstraFoundation/substra/blob/master/examples/titanic/README.md).
+
+Please note that each resource is referenced by its hash in Substra, a unique identifier, so if you want to try out the ready to use Titanic example, you will need to edit, at least, the python files and the `description.md` for the assets (like adding a `print()` statement and editing the `description.md` files).
```

Comment:
I don't understand this part. Is this optional? What would be the goal of adding a `print()` statement and where exactly? This part is really confusing.

Reply:
I should not be required with the `exist_ok` option, I wanted to confirm this but I can get any running instance...